src/sentry/monitors/models.py

```diff
@@ -604,11 +629,11 @@ class MonitorIncident(Model):
     monitor_environment = FlexibleForeignKey("sentry.MonitorEnvironment")
     starting_checkin = FlexibleForeignKey(
         "sentry.MonitorCheckIn", null=True, related_name="created_incidents"
-    )
+    )  # this represents the first failed check-in that we receive
     starting_timestamp = models.DateTimeField(null=True)
```

Comment:
Let's write this as a doc string

```suggestion
    )
    starting_timestamp = models.DateTimeField(null=True)
    """
    This represents the first failed check-in that we receive
    """

```